recipes/libyuv/all/conanfile.py

```diff
@@ -48,11 +48,11 @@ def layout(self):
 
     def requirements(self):
         if self.options.with_jpeg == "libjpeg":
-            self.requires("libjpeg/9e")
+            self.requires("libjpeg/9f")
         elif self.options.with_jpeg == "libjpeg-turbo":
-            self.requires("libjpeg-turbo/3.0.1")
+            self.requires("libjpeg-turbo/3.0.3")
```

Comment:
Consider using version range for some of this libraries.
I have to discuss with the team